sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1691,19 +1692,53 @@ def plot_loading_results(self, subs=None):
 
         return PlotGrid(4, 1, ax1, ax2, ax3, ax4)
 
-    def _solve_for_ild_equations(self):
+    def _solve_for_ild_equations(self, ild_load_value, *reactions):
```

Comment:
Add something in the docs below for these parameters. Maybe a small definition for each.